### DISCUSSION TITLE
activities.0.currency ("EUR") does not match with "USD" while importing CSV

### DISCUSSION THREAD

**Discussion by JeeSeeNL** [ghostfolio/ghostfolio#1975] [Q&A]
I'm trying to import activities from .CSV with other currency than 'USD'.
Following error occurs:
"activities.0.currency ("EUR") does not match with "USD"

I build my .CSV like the example _OK.CSV_ but with Currency EUR

Following error occurs:
"activities.0.currency ("EUR") does not match with "USD"

BASE_CURRENCY of installation is set to "EUR", and this value is also set in profile.

Manual activity entering in currency EUR is going well.

- Ghostfolio v1.269.0, self-hosted
- Chrome browser

**Comment by dtslvr** [comment]
Hi @JeeSeeNL

What activity are you trying to import? As background information, the currency of the symbol (given by the exchange) must match the currency of your activity. Otherwise there is this mismatch.

**Comment by JeeSeeNL** [comment]
I'm trying to import transactions by the following format in .CSV.:

Date;Action;Code;Quantity;Price;Currency;Fee
26-4-2023;buy;BTC;0.00203394;27057;EUR;0.13768542
26-4-2023;buy;BTC;0.00165305;27050;EUR;0.1149975
25-4-2023;buy;BTC;0.00403114;24745;EUR;0.2494407

Am I missing something?

**Reply by dtslvr** [reply]
Here is what happens behind the scene:

If you leave out the column `dataSource`, it will fallback to `YAHOO`. The `code` (symbol) is defined as `BTC`, according to [_Yahoo Finance_](https://finance.yahoo.com/quote/BTC) this ETF (_ClearShares Piton Intermediate Fixed Income ETF_) is traded in `USD` and you are providing the currency `EUR`, thus the mismatch.

**Comment by JeeSeeNL** [comment]
Ehhm.. I don't think I understand. I'm trying to import CRYPTO transactions, and BTC stands for BITCOIN, isn't it? 
If I'm doing something wrong, how can I import my CRYPTO transactions the right way?

**Comment by dtslvr** [comment]
You can try to import a `csv` file with the following content:

```
Date,Code,Currency,Price,Quantity,Action,Fee
08/06/2023,BTCUSD,USD,26569.20,1,buy,0.00
```

**Comment by JeeSeeNL** [comment]
Well, almost there, but now the quantity column is zero...

![ghostfolio quantity empty](https://github.com/ghostfolio/ghostfolio/assets/93616327/5e76e1d7-1d7f-4a22-9bb6-4c7fa5efeb67)
